site/content/en/docs/tasks/troubleshooting/troubleshooting_provreq.md

```diff
@@ -0,0 +1,102 @@
+
+One of the reason your job is not running is that ProvisioningRequest is waiting to be provisioned. To find out if this is the case you can view Provisioning Request's state by running the following command:
+
+```bash
+kubectl get provisioningrequest PROVISIONING_REQUEST_NAME
+```
+
+and more detailed version with a command:
+
+```bash
+kubectl describe provisioningrequest PROVISIONING_REQUEST_NAME
+```
+
+Provisioning Request state is described in its Status in the `Conditions` field.  it means it still being processed by the Cluster Autoscaler. Otherwise it falls into one the states listed below:
+- Accepted - indicates that the ProvisioningRequest was accepted by ClusterAutoscaler, so ClusterAutoscaler will attempt to provision the nodes for it.
+- Provisioned - indicates that all of the requested resources were created and are available in the cluster. CA will set this condition when the VM creation finishes successfully.
+- Failed - indicates that it is impossible to obtain resources to fulfill this ProvisioningRequest.	Condition Reason and Message will contain more details about what failed.
+- BookingExpired - indicates that the ProvisioningRequest had Provisioned condition before and capacity reservation time is expired.
+- CapacityRevoked - indicates that requested resources are not longer valid.
+
+The states transitions are as follow:
+
+![Provisioning Request's states](/images/prov-req-states.svg)
+
+## Provisioning Request is not created by Kueue
+
+If your job does not create a corresponding Provisioning Request try checking following requirements:
+
+### Ensure the Kueue's controller manager enables the `ProvisioningACC` feature gate
+
+Run a following command to check whether your Kueue's controller manager enables the `ProvisioningACC` feature gate:
+
+```bash
+kubectl describe pod -n kueue-system kueue-controller-manager-
+```
+
+The args for kueue container should be similar to this:
+
+```bash
+    ...
+    Args:
+      --config=/controller_manager_config.yaml
+      --zap-log-level=2
+      --feature-gates=ProvisioningACC=true
+```
+
+### Ensure the ClusterQueue and the LocalQueue are active
+
+For details see our other [troubleshooting guide](/docs/tasks/troubleshooting/troubleshooting_queues/)
+
+
+### Ensure the Admission Check is active
+
+To check if the Admission Check your job uses is active run the following command:
+
+```bash
+kubectl describe admissionchecks ADMISSIONCHECK_NAME
+```
+
+The status of the Admission Check should be similar to:
+
+```bash
+...
+Status:
+  Conditions:
+    Last Transition Time:  2024-03-08T11:44:53Z
+    Message:               The admission check is active
+    Reason:                Active
+    Status:                True
+    Type:                  Active
+```
```

Comment:
What if all of steps above works but still no ProvReq? 
Maybe worth to file a bug in the repo or ask in slack?

Reply:
Fixed